Samples/auth/Outlook-Add-in-SSO-NAA-Identity/README.md

```diff
@@ -0,0 +1,276 @@
+1. In the app's registration screen, select the **Authentication** blade on the left pane.
+1. If you don't have a platform added, select **Add a platform** and choose the **Single-page application** option.
+
+    ![select redirect platform](./images/add-platform.png)
+
+    1. In the **Redirect URI** section, enter the following redirect URIs:
+        1. `http://localhost:3000`
+        1. `http://localhost:3000/redirect`
+        1. `brk-multihub://localhost:3000`
+    1. To save your changes, select **Save**.
+
+    ![redirect uris](./images/redirect-uris-completed.png)
+
+Since this app signs-in users, you'll now proceed to select **delegated permissions**, which is required by apps signing-in users.
+
+1. In the app's registration screen, select the **API permissions** blade on the left pane. This opens the page where you add access to the APIs that your application needs.
+1. Select the **Add a permission** button, and choose the **APIs my organization uses** tab.
+1. In the list of APIs, choose the API `Contoso-Web-API-Server`.
+  
+    ![select permissions](./images/select-permissions.png)
+
+1. In the **Delegated permissions** section, select **Todolist.Read**, **Todolist.ReadWrite** in the list. Use the search box if necessary.
+1. Select the **Add permissions** button.
+
+Your scopes should appear as shown in the following screenshot.
+
+![list of scopes](./images/scopes-added-client.png)
+
+### Configure the client app (Contoso-Outlook-Add-in) to use your app registration
+
+Open the sample project in Visual Studio Code to configure the code.
+
+1. Open the `SPA/src/taskpane/msalconfig.ts` file.
+1. Find the key `Enter_Application_Client_Id_Here` and replace the existing value with the application ID (clientId) of `Contoso-Outlook-Add-in` app copied from the Microsoft Azure app registration page.
+1. Save the file.
+
+Next you need to add the API web server's application ID. **This is not the same ID as the previous steps**.
+
+1. Open the `SPA/src/taskpane/authConfig.ts` file.
+1. Find the key `Enter_API_Application_Id_Here` and replace the existing value with the application ID (clientId) of `Contoso-Web-API-Server` app copied from the Microsoft Azure app registration page.
+
+## Run the sample
+
+First, start the web API service. In a console window, go to the root project folder of the sample. Run the following commands.
+
+```console
+    cd API
+    npm install
+    npm start
+```
+
+Next, start and sideload the Outlook add-in. In a new console window, go to the root project folder of the sample. Run the following commands.
+
+```console
+    cd SPA
+    npm install
+    npm start
+```
+
+The following instructions assume you are sideloading in Outlook for Windows. To sideload on other platforms, see [Sideload Outlook add-ins for testing](https://learn.microsoft.com/office/dev/add-ins/outlook/sideload-outlook-add-ins-for-testing)
```

Comment:
The following instructions don't talk about sideloading - they assume you've already sideloaded.

Reply:
fixed.